components/SearchSandbox/components/RSWrapper.js

```diff
@@ -80,14 +80,17 @@ export default class RSWrapper extends Component {
 		});
 	}
 
-	getAvailableDataField = (fieldType) => {
-		const { types } = propsMap[this.props.component][fieldType];
+	getAvailableDataField = () => {
+		let types = propsMap[this.props.component].dataField.types;
 
-		if (this.props.id === 'search' && fieldType !== 'categoryField') {
-			return Object.keys(this.props.mappings).filter(field =>
-				types.includes(this.props.mappings[field].type));
+		if(this.props.component === 'CategorySearch'){
+			types = propsMap[this.props.component].categoryField.types;
 		}
 
+
+		if (this.props.component === 'DataSearch') {
```

Comment:
Why are we doing these checks? They should both return the same fields for `dataField` prop.